ray_on_golem/provider/node_provider.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, provider_config: dict, cluster_name: str):
         self._ray_on_golem_client.create_cluster(
             network=provider_parameters["network"],
             budget_limit=provider_parameters["budget_limit"],
-            node_config=NodeConfigData(**provider_parameters["node_config"]),
+            node_config=NodeConfigData(**(provider_parameters.get("node_config") or {})),
```

Comment:
```suggestion
            node_config=NodeConfigData(**(provider_parameters.get("node_config", {}))),
```

Reply:
It won't work if `provider_parameters` has `"node_config":None`